packages/ti_eclecticiq/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "1.0.0"
```

```suggestion
- version: "0.1.0"
```

We usually launch new integrations with `0.x.0` which indicates `beta` integration. After few months and enough usage, we make it to `1.0.0` i.e., `GA`.